css/css-pseudo/highlight-cascade-001.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>CSS Pseudo-Elements Test: highlight cascade: unset is treated like inherit regardless of whether property is inherited</title>
```

Comment:
Might be interesting to add similar tests for `inherit` and `revert`.

Reply:
can you expand on what you had in mind? with unset, the value is defined as equivalent to inherit or initial depending on the property, making it interesting to test what counts as an “inherited property” (because all properties are inherited in highlight styles). but for inherit and revert, there’s no direct counterpart behaviour that depends on inheritedness.

Reply:
Maybe test that `inherit` inherits from the highlight of the parent element of the originating element, rather than from the originating element. And for `revert` test how it plays with

> The UA must use its own highlight colors for ''::selection'' only when neither 'color' nor 'background-color' has been specified by the author

But maybe there are already tests for that. Or you can add them in a follow-up. Not blocking this.

Reply:
thanks! i’ll check if there are tests for those or otherwise add in another patch.